ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -253,18 +255,24 @@ def get_workflow(dict_configs, cdump='gdas'):
     do_wafs = base.get('WAFSF', 'NO').upper()
     do_vrfy = base.get('DO_VRFY', 'YES').upper()
     do_metp = base.get('DO_METP', 'NO').upper()
+    datm_src = base.get('datm_src', 'gefs').lower()
     n_tiles = 6
 
     tasks = []
 
-    if 'S2S' in app:
-        # Copy prototype ICs
-        deps = []
+    if app in ['S2S', 'S2SW', 'OCN-ICE']:
```

Comment:
This won't work the same as before (no S2SWA). Please change to `if 'S2S' in app or app in ['OCN-ICE']`